docs/develop/sso-in-office-add-ins.md

```diff
@@ -34,7 +34,7 @@ The following diagram shows how the SSO process works.
 
 ![A diagram that shows the SSO process](../images/sso-overview-diagram.png)
 
-1. In the add-in, JavaScript calls a new Office.js API [getAccessToken](/javascript/api/office/officeruntime.auth#getAccessToken-options--callback-). This tells the Office host application to obtain an access token to the add-in. See [Example access token](#example-access-token).
+1. In the add-in, JavaScript calls a new Office.js API [getAccessToken](/javascript/api/office-runtime/officeruntime.auth#getAccessToken-options-). This tells the Office host application to obtain an access token to the add-in. See [Example access token](#example-access-token).
```

Comment:
bookmark portion of the link target needs to be all lowercase -> `#getaccesstoken-options-`